src/test/java/com/cloudbees/jenkins/support/impl/RunningJobsTest.java

```diff
@@ -0,0 +1,45 @@
+package com.cloudbees.jenkins.support.impl;
+
+import com.cloudbees.jenkins.support.SupportTestUtils;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.queue.QueueTaskFuture;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertThat;
+
+public class RunningJobsTest {
+
+    private static final String JOB_NAME = "job-name";
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    public void addContents() throws Exception {
+        FreeStyleProject p = j.createFreeStyleProject(JOB_NAME);
+        QueueTaskFuture<FreeStyleBuild> build = p.scheduleBuild2(0);
+        build.waitForStart();
+
+        String output = SupportTestUtils.invokeComponentToString(new RunningJobs());
+
+        assertThat(output, containsString(p.getName()));
```

Comment:
Could perhaps be more specific:
```suggestion
        assertThat(output, containsString(p.getName() + "#1"));
```

or whatever it is.

(BTW not a bad idea to use `JenkinsRule.createFolder` and assert that content contains the full path.)